### PR TITLE
Release 2.5.0

### PR DESCRIPTION
**Pull request by elgatovital**
@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt [hunk 1]
```diff
@@ -68,7 +68,7 @@ class TransactionDetailsActionMultisendFragment : BaseViewBindingFragment<Fragme
                                 value.data,
                                 value.dataDecoded?.let { paramSerializer.serializeDecodedData(it) },
                                 value.to.asEthereumAddressString(),
-                                balanceFormatter.formatAmount(value.value, true)
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt:71]: Why is this different on master and release? And is false the right value for incoming here?

File: app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt [hunk 2]
```diff
@@ -80,8 +80,10 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
 
         binding.transactions.addOnScrollListener(object : RecyclerView.OnScrollListener() {
             override fun onScrolled(recyclerView: RecyclerView, dx: Int, dy: Int) {
-                val firstVisibleItem = (binding.transactions.layoutManager as LinearLayoutManager).findFirstVisibleItemPosition()
-                reload = firstVisibleItem <= 1
+                if (lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt:83]: I guess checking the lifecycle state is the right thing to do, but why is this not already on master?

Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt:83]: this was relevant for release, so it was fixed here, to produce new rc

File: app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/transactions/details/TransactionDetailsActionMultisendFragment.kt:71]: https://app.zenhub.com/workspaces/safe-multisig---mobile-5c1b98250e13551e8aae81eb/issues/gnosis/safe-android/1041 was fixed directly on release to produce new rc

**Review by biafra23 — APPROVED** [review]

**Pull request merged**